examples/cms-wordpress/next.config.js

```diff
@@ -8,7 +8,8 @@ if (!process.env.WORDPRESS_API_URL) {
 module.exports = {
   images: {
     domains: [
-      process.env.WORDPRESS_API_URL.match(/(http(?:s)?:\/\/)(.*)/)[2], // Valid WP Image domain.
+      process.env.WORDPRESS_API_URL.match(/http(?:s)?:\/\/(.[^/]+)/)[1], // Valid WP Image domain.
```

Comment:
Why did this change?

Reply:
For example if we have `WORDPRESS_API_URL  = "http://127.0.0.1/wordpress/graphql"`
When we using `match(/http(?:s)?:\/\/(.[^/]+)/)` 

## **Actual Result**

`WORDPRESS_API_URL.match(/http(?:s)?:\/\/(.[^/]+)/)[2] // 127.0.0.1/wordpress/graphql` 
Which is not valid WP image domain so `Next` will throw error on `<Image/>` component
![exciting-mcnulty-bi5ylq - CodeSandbox - Google Chrome_2](https://user-images.githubusercontent.com/30198386/182752008-fd2bc066-b2d9-4ebf-9281-40691dbd822f.jpg)

## **Expected Result**  

- 127.0.0.1
- example.com `(if WORDPRESS_API_URL = "http://example.com/wordpress/graphql" or "http://www.example.com/wordpress/graphql")` currently this regex not extract www from url will fix this in next commit